src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7784,4 +7784,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_TargetDifferentRefness_Title" xml:space="preserve">
     <value>Reference kind modifier of parameter doesn't match the corresponding parameter in target.</value>
   </data>
+  <data name="WRN_UseDefViolationRefField" xml:space="preserve">
+    <value>ref field '{0}' should be ref-assigned before use.</value>
```

Comment:
Should the warning sentence start with an uppercase letter?

Reply:
I don't know. I thought it's better this way because it looks more like the ref modifier.

Reply:
```
  <data name="ERR_RefReturningPropertiesCannotBeRequired" xml:space="preserve">
    <value>Ref returning properties cannot be required.</value>
  </data>
```

seems like decent enough precedent to change the casing.